README.md

```diff
@@ -32,6 +32,14 @@ Discord server).
 
 1. Install from PyPI:
 
+    Install a torch version that fulfills your needs:
+
+    ```bash
+    # Fast installation of torch with just CPU support.
+    # See other options at https://pytorch.org/get-started/locally/
+    pip install -r pytorch-cpu-requirements.txt
+    ```
```

Comment:
These user instructions don't assume that you have the repository checked out yet, so I don't think we should depend on this file (and it only works on Linux right now)

Reply:
Good catch. Updated again.

Reply:
Oh, you updated the section below but not this first user section? The text down there LGTM, but this user section right at the top of the docs shouldn't assume you have the git repo cloned.